File: src/test/java/com/csl9015/PersonTest.java
package com.csl9015;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class PersonTest {
    @Test
    public void bmiTest() {
        Person person = new Person(66.5f, 1.7f);
        Assertions.assertEquals(66.5f / (1.7f * 1.7f), person.bmi());
    }

    // Kotlin
    @Test
    public void bmikTest() {
        Human human = new Human(66.5f, 1.7f, "Willy");
        Assertions.assertEquals(66.5f / (1.7f * 1.7f), human.bmi());
    }
}

File: src/test/kotlin/com/csl9015/PersonTest.kt
package com.csl9015.kotlin

import com.csl9015.Human
import com.csl9015.Person
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class PersonTest {
    @Test
    fun humanBmiTest() {
        val human = Human(66.5f, 1.7f)
        Assertions.assertEquals(66.5f / (1.7f * 1.7f), human.bmi())
    }

    // Java
    @Test
    fun personBmiTest() {
        val person = Person(66.5f, 1.7f)
        Assertions.assertEquals(66.5f / (1.7f * 1.7f), person.bmi())
    }
}


